src/logic/safe/transactions/gas.ts

```diff
@@ -130,7 +130,7 @@ export const getDataFromNodeErrorMessage = (errorMessage: string): string | unde
   }
 }
 
-const getGasEstimationTxResponse = async (txConfig: {
+export const getGasEstimationTxResponse = async (txConfig: {
```

Comment:
Is this related with this feature? Would it cause conflicts?

Reply:
I forget to remove this thanks for noticing it, this change is not needed